local-cli/runAndroid/runAndroid.js

```diff
@@ -123,15 +123,28 @@ function buildAndRun(args, reject) {
 }
 
 function startServerInNewWindow() {
+  var yargV = require('yargs').argv;
+
```

Comment:
Why yargs is not being used in cli.js is beyond me when it is already a dependency.